config/locales/en.yml

```diff
@@ -3,6 +3,12 @@ en:
   service:
     name: Submit a crime form
   shared:
+    plea_category:
```

Comment:
Since we now have these in the gem, can we avoid duplication and use the gem translations?

Reply:
yeah this is just placeholder - was waiting for my update to the common gem to be approved

Reply:
You can just update the `Gemfile` to point to a local path instead

```diff
@@ -13,7 +13,9 @@ gem 'govuk_notify_rails', '~> 3.0.0'
 gem 'grover'
 gem 'httparty'
 gem 'jsbundling-rails'
-gem 'laa_crime_forms_common', '~> 0.3.0', github: 'ministryofjustice/laa-crime-forms-common'
+# gem 'laa_crime_forms_common', '~> 0.3.0', github: 'ministryofjustice/laa-crime-forms-common'
+# TODO: Remove this before merge
+gem 'laa_crime_forms_common', path: '../laa-crime-forms-common'
 gem 'laa_multi_step_forms', path: './gems/laa_multi_step_forms'

```